dbt_coverage/__init__.py

```diff
@@ -225,6 +202,27 @@ def from_nodes(cls, manifest_nodes: Dict[str, Dict]) -> Manifest:
 
         return Manifest(sources, models, seeds, snapshots, tests)
 
+    def get_table(self, table_id):
+        source_candidate = self.sources.get(table_id)
+        models_candidate = self.models.get(table_id)
+        seeds_candidate = self.seeds.get(table_id)
+        snapshots_candidate = self.snapshots.get(table_id)
+
+        all_candidates = [
+            source_candidate,
+            models_candidate,
+            seeds_candidate,
+            snapshots_candidate,
+        ]
+        not_empty_candidates = [c for c in all_candidates if c]
```

Comment:
@sweco I was expecting non_empty here but it turns out not_empty is proper english too :)

https://www.italki.com/en/post/question-279576

Reply:
However, they say that `non_empty` is preferred. Will change that! 😊